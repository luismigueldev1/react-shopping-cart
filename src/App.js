import React from "react";
import AppRouter from "./routes/AppRouter";
import StoreProvider from "./store/StoreProvider";

function App() {
  return (
    <StoreProvider>
      <AppRouter />
    </StoreProvider>
  );
}

export default App;
