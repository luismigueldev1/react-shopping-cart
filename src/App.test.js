import { mount } from "enzyme";
import App from "./App";

describe("Pruebas en <App/>", () => {
  const wrapper = mount(<App />);
  test("Debe de mostrarse correctamente ", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
