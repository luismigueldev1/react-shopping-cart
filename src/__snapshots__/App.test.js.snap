// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <App/> Debe de mostrarse correctamente  1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <div
      className="container-fluid"
    >
      <a
        className="navbar-brand "
        href="/"
        onClick={[Function]}
      >
        Mi tienda ReactJS
      </a>
      <button
        aria-controls="navbarNavAltMarkup"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-bs-target="#navbarNavAltMarkup"
        data-bs-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse"
        id="navbarNavAltMarkup"
      >
        <div
          className="navbar-nav me-auto"
        />
        <ul
          className="navbar-nav"
        >
          <li
            className="nav-item nav-link"
          >
            <span
              className="text-white"
            />
          </li>
          <li
            className="nav-item"
          />
        </ul>
      </div>
    </div>
  </nav>
  <div
    className="container"
  >
    <div
      className="row justify-content-center align-items-center"
    >
      <div
        className="col-4  d-flex flex-column mt-5"
      >
        <h3
          className="text-center"
        >
          Autenticaci√≥n
        </h3>
        <button
          className="btn btn-primary  mt-3"
          onClick={[Function]}
        >
          Registrarse
        </button>
        <button
          className="btn btn-secondary  mt-3"
          onClick={[Function]}
        >
          Invitado
        </button>
      </div>
    </div>
  </div>
</div>
`;
